<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Person form</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Person Data:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Gender</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QRadioButton" name="radioButton">
                <property name="text">
                 <string>Male</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_2">
                <property name="text">
                 <string>Female</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_3">
                <property name="text">
                 <string>Other</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Age</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ageEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Skin color</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <property name="currentText">
              <string>Light</string>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
             <item>
              <property name="text">
               <string>Light</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Medium</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Dark</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Eye color</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_3">
             <property name="currentText">
              <string>Brown</string>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
             <item>
              <property name="text">
               <string>Brown</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Green</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Blue</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Hair color</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_4">
             <property name="currentText">
              <string>Light</string>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
             <item>
              <property name="text">
               <string>Light</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Dark</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Face shape</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="faceShapeEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Eye shape</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="eyeShapeEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Face features</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="faceFeaturesEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Body features</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="bodyFeaturesEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>Social status</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="socialStatusEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="nextPageButton">
             <property name="text">
              <string>Next page</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Body Data:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Weight</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="weightEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Height</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="heightEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>Chest girth</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="chestGirthEdit">
             <property name="inputMethodHints">
              <set>Qt::ImhNone</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>Waist girth</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="waistGirthEdit">
             <property name="inputMethodHints">
              <set>Qt::ImhNone</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>Hip girth</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="hipGirthEdit">
             <property name="inputMethodHints">
              <set>Qt::ImhNone</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Body fat</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="bodyfatEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Muscle</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="muscleEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Water</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="waterEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Protein</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="proteinEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Visceral fat</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="visceralFatEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Basal metabolism</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="basalMetabolismEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>BMI</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="bmiEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Body type</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="bodyTypeCombobox">
             <property name="currentText">
              <string>Under weight</string>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
             <item>
              <property name="text">
               <string>Under weight</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Normal weight</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Over weight</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Obese</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="prevPageButton">
             <property name="text">
              <string>Previous page</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Submit</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
